### DISCUSSION TITLE
SAI Support

### DISCUSSION THREAD

**Discussion by samersitc** [nosan/embedded-cassandra#456] [Q&A] [answered]
Hi,
does embedded-cassandra 5.0.0 support the SAI functionality?

**Comment by nosan** [comment, marked as answer]
Hi,
I haven't tested it yet, but it should be supported by default. Embedded Cassandra downloads and runs Apache Cassandra directly, so if Apache Cassandra 5.0.0 includes support for SAI, it should work the same way with Embedded Cassandra.

**Reply by samersitc** [reply]
Yes, it does support SAI :)
Thanks
